match/anything.go

```diff
@@ -0,0 +1,48 @@
+package match
+
+import "reflect"
+
+// Anything returns a new matcher that will match any value
+func Anything() SupportedKindsMatcher {
+	return &anything{}
+}
+
+type anything struct {
+}
+
+// SupportedKinds returns all the kinds the anything matcher supports
+func (m *anything) SupportedKinds() map[reflect.Kind]struct{} {
+	return map[reflect.Kind]struct{}{
+		reflect.Bool:          struct{}{},
```

Comment:
`struct{}` can be removed from all these initializers. this file and all other `match` files.